exercises/phone-number/canonical-data.json

```diff
@@ -72,15 +72,37 @@
         },
         {
           "description": "invalid if area code does not start with 2-9",
-          "property": "clean",
-          "phrase": "(123) 456-7890",
-          "expected": null
+          "cases": [
+            {
+              "description": "invalid if area code starts with 0",
+              "property": "clean",
+              "phrase": "(023) 456-7890",
+              "expected": null
+            },
+            {
+              "description": "invalid if area code starts with 1",
+              "property": "clean",
+              "phrase": "(123) 456-7890",
+              "expected": null
+            }
+          ]
         },
         {
           "description": "invalid if exchange code does not start with 2-9",
-          "property": "clean",
-          "phrase": "(223) 056-7890",
-          "expected": null
+          "cases": [
+            {
+              "description": "invalid if exchange code starts with 0",
+              "property": "clean",
+              "phrase": "(223) 056-7890",
+              "expected": null
+            },
+            {
+              "description": "invalid if exchange code starts with 1",
+              "property": "clean",
+              "phrase": "(223) 156-7890",
+              "expected": null
+            }
+          ]
```

Comment:
Ditto here. Prefer two new invalid cases, but without another nested level of cases.